zerver/lib/user_status.py

```diff
@@ -113,3 +113,22 @@ def update_user_status(
         user_profile_id=user_profile_id,
         defaults=defaults,
     )
+
+
+def get_user_status(user_profile_id: int) -> UserInfoDict:
```

Comment:
It might be worth a prep commit to rename `get_user_status_dict` so that it's clearer that function returns the status of all users in the realm. That way the distinction between that function and this new one would be more obvious.

Also, noted that we could probably just pass the UserProfile to this function.

Reply:
I have renamed `get_user_status_dict` -> `get_all_users_status_dict`.